webapi/src/main/scala/org/knora/webapi/responders/v2/SearchResponderV2.scala

```diff
@@ -231,13 +230,13 @@ class SearchResponderV2 extends ResponderWithStandoffV2 {
                 case iriRef: IriRef =>
                     // type info keys are external api v2 simple Iris,
                     // so convert this internal Iri to an external api v2 simple if possible
```

Comment:
we should get rid of this comment because we do not convert to internal anymore, right?

Reply:
Do you mean just get rid of the comment, or also get rid of the conversion from internal to external?